custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -730,7 +732,17 @@ def _run_inside_docker(self, options, run_mode, raw_arguments):
 
         self._print_verbose("Checking DRUM version in container...")
         result = subprocess.run(
-            ["docker", "run", "-it", options.docker, "sh", "-c", "drum --version"],
+            [
+                "docker",
+                "run",
+                "-it",
+                "--entrypoint",
+                "",
```

Comment:
what's this empty arg for?

Reply:
+1

Reply:
Passing empty entrypoint
`--entrypoint ""`

Reply:
Unsetting if one is set within the image

Reply:
might be worth leaving a comment

Reply:
done